WebRoot/js/zimbraMail/mail/view/prefs/ZmAccountsPage.js

```diff
@@ -1974,13 +1979,27 @@ function(id, section, value) {
 			control.setSelected(value);
 			break;
 		}
-		case ZmPref.TYPE_INPUT:
+		case ZmPref.TYPE_INPUT: {
+			if (setup.ariaLabel) {
+				control._inputField.setAttribute('aria-label', setup.ariaLabel);
+			}
+			control.setValue(value);
+			break;
+		}
 		case ZmPref.TYPE_COMBOBOX: {
+			if (setup.ariaLabel) {
+				control.input._inputField.setAttribute('aria-label', setup.ariaLabel);
+				control._button.setAriaLabel(setup.ariaLabel);
+				control._button.setAttribute('aria-expanded', false);
```

Comment:
will it help to create setAriaLabel method and move this entire logic to there?

Reply:
Done.